Source/Csla/Csla.csproj

```diff
@@ -12,6 +12,7 @@
     <BaseOutputPath>..\..\Bin</BaseOutputPath>
     <GenerateDocumentationFile>True</GenerateDocumentationFile>
     <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
+    <Version>9.0.0.5</Version>
```

Comment:
There should be no `Version` element here - it comes from Directory.build.props

Reply:
I will remove it